client/app/lib/views/credentiallist/accountcredentiallist.coffee

```diff
@@ -24,14 +24,22 @@ module.exports = class AccountCredentialList extends KodingListView
   showCredential: (options = {}) ->
 
     { credential, cred } = options
+    
+    content = "<p><pre><code>" + cred + "</code></pre></p>"
```

Comment:
https://github.com/koding/styleguide-coffeescript#use-string-interpolations-instead-of-string-concatenation

`content = "<p><pre><code>#{cred}</code></pre></p>"`